packages/blockchain-wallet-v4-frontend/src/data/components/interest/actions.ts

```diff
@@ -275,9 +276,14 @@ export const showInterestModal = (
   type: AT.SHOW_INTEREST_MODAL
 })
 
+export const stopShowingInterestModal = () => ({
+  type: AT.STOP_SHOWING_INTEREST_MODAL
+})
+
 // INTEREST CTA AFTER TRANSACTION
-export const fetchAfterTransaction = () => ({
-  type: AT.FETCH_AFTER_TRANSACTION
+export const fetchAfterTransaction = (currency?: WalletFiatType) => ({
```

Comment:
can these functions, actions, actionTypes be named something more descriptive? `fetchShowInterestCardAfterTransaction` or something of the like

Reply:
Yeah will change it, I use naming based on API endpoint, but you are right